CHANGELOG.md

```diff
@@ -1,3 +1,17 @@
+## v0.130.0 [2024-02-02]
+
+_Enhancements_
+
+- Improved filter param by adding new key columns `descending`, `log_group_name`, `log_stream_name_prefix` and `order_by` and added new columns `log_stream_name_prefix`, `descending` and `order_by` in table `aws_cloudwatch_log_stream`. ([#1951](https://github.com/turbot/steampipe-plugin-aws/pull/1951))
+- Improved filter param of the table `aws_ssm_inventory` by adding new key columns. ([#1980](https://github.com/turbot/steampipe-plugin-aws/pull/1980))
```

Comment:
@khushboo9024 could you please mention which columns were added and update the note as per the above suggestion?